test/geometry/GeoParams.test.cc

```diff
@@ -57,6 +69,16 @@ TEST_F(GeoParamsHostTest, print_geometry)
     auto device_view = this->params()->device_pointers();
     EXPECT_NE(nullptr, device_view.world_volume);
 
+    const auto& geom = *(this->params());
+    EXPECT_EQ(11, geom.num_volumes());
+    EXPECT_EQ(4, geom.max_depth());
+    EXPECT_EQ("Envelope",
+              geom.id_to_label(VolumeId{
+                  static_cast<unsigned int>(geom.num_volumes()) - 2}));
+    EXPECT_EQ("World",
+              geom.id_to_label(VolumeId{
+                  static_cast<unsigned int>(geom.num_volumes()) - 1}));
+
```

Comment:
These tests look like duplicates of the ones above.

Reply:
True.  Removed.  What is left is basically a potentially long printout of the geometry elements from the GPU, quite useful for debugging, but not for a unit test, so I removed the whole test.